apps/web/public/static/locales/en/common.json

```diff
@@ -2682,6 +2682,7 @@
   "booking_limits_updated_successfully": "Booking limits updated successfully",
   "you_are_unauthorized_to_make_this_change_to_the_booking": "You are unauthorized to make this change to the booking",
   "matching_members": "Matching members",
+  "x_matching_members": "{{x}} matching members",
```

Comment:
NIT: probably a good idea to use context to confirm the type of X here for our translation contributors.